great_expectations/profiler/parameter_builder/multi_batch_bootstrapped_metric_distribution_parameter_builder.py

```diff
@@ -71,7 +71,11 @@ def _build_parameters(
             metric_domain_kwargs = copy(rule_state.active_domain["domain_kwargs"])
             metric_domain_kwargs.update({"batch_id": batch_id})
 
-            if self._metric_value_kwargs.startswith("$"):
+            if (
+                self._metric_value_kwargs
+                and isinstance(self._metric_value_kwargs, str)
```

Comment:
Making metric_value_kwargs a string does not make any sense - if this is the case, why are they called kwargs

Reply:
@gilpasternak35 I agree with you.  This came from the prototype, and is an overloading of types (we see this kind of a shortcut here and there, and, I believe you already commented on such a case earlier).  I would pose a question to the team: do we ever express the `metric_value_kwargs` as if it is, in reality, a `$metric.value.string`? :-). Let us discuss tomorrow, then resolve, and makes any necessary changes accordingly. /cc @talagluck @anthonyburdi

Reply:
@gilpasternak35 Just now, @anthonyburdi and I discussed this thoroughly and agreed that this coding pattern has to be fixed (among other potential issues with this class).  For now, we agreed that I will heavily annotate this class with our findings, but defer the changes till later when we have concrete use cases for it.  Thank you for flagging this!

Reply:
As discussed, we should process these $string variables elsewhere (parent class probably).